### PR TITLE
Bottom text added, Fixes #168

### LINKED ISSUE #168: Add bottom text, address and Github icon
Please do the following changes on all pages:
* On the top right of the menu add a Github icon and link to https://github.com/fossasia
* On the footer change the FOSSASIA text as follows:
>  About FOSSASIA
FOSSASIA develops Open Source software and hardware solutions with a global developer community from its base in Asia and organizes Open Technology events around the year. The organization runs coding programs and development contests. The annual FOSSASIA Summit is the premier Open Technology event in Asia for developers, contributors, start-ups, and technology companies. FOSSASIA was founded in 2009 by Mario Behling and Hong Phuc Dang.

* On the footer change the address to: 
> Ms. DANG Hong Phuc
FOSSASIA
6 Eu Tong Sen Street, #05-07 The Central, Singapore 059817

### PR DESCRIPTION
**Pull request by Pk13055**
- Small changes included unicode `and` and other symbols.
- Bottom text changed for the introduction of FossAsia in the footer.
- Github links added for the nav bar
- Address also changed

### CODE REVIEW

**Comment by Pk13055** [comment]
#168

**Review by mariobehling — APPROVED** [review]

File: apply.html [hunk 1]
```diff
@@ -118,6 +118,7 @@
 									<li class="hidden-sm hidden-xs"><a href="http://blog.fossasia.org/contact/" target="_self">Contact</a></li>											
 									<li class="social-link hidden-md hidden-sm hidden-xs"><a target="_self" href="https://twitter.com/fossasia"><i class="icon social_twitter"></i></a></li>
 									<li class="social-link hidden-md hidden-sm hidden-xs"><a target="_self" href="https://facebook.com/fossasia"><i class="icon social_facebook"></i></a></li>
+									<li class="social-link hidden-md hidden-sm hidden-xs"><a target="_self" href="https://github.com/fossasia"><i class="fa fa-github"></i></a></li>
```
Comment (Pk13055) [inline apply.html:121]: @mariobehling Isn't this the icon you are talking about in #168?

File: licenses.html [hunk 2]
```diff
@@ -249,14 +249,14 @@ <h1 class="text-white">FOSSASIA License Agreements</h1>
...
 						</div>
 
 						<div class="col-sm-3">
 							<ul class="contact-methods">
 								<li><i class="icon pe-7s-mail"></i><span><a href="mailto:fossasia@googlegroups.com" target="_self" style="color:#777777">fossasia@googlegroups.com</a></span></li>
 								<li><i class="icon pe-7s-phone"></i><span>+65 83435672</span></li>
-								<li><i class="icon pe-7s-map-marker"></i><span>Ms. DANG Hong Phuc</span><span>FOSSASIA Xoai Building, 93 Mau Than Street Xuan Khan, Ninh Kieu 9000 Can Tho, Vietnam, Asia</span></li>
+								<li><i class="icon pe-7s-map-marker"></i><span>Ms. DANG Hong Phuc</span><span>6 Eu Tong Sen Street #05-07 The Central, Singapore 059817</span></li>
```
Comment (Pk13055) [inline licenses.html:259]: So basically there should be an `li` with _FOSSASIA_ before the updated address? @mariobehling

Comment (mariobehling) [reply licenses.html:259]: Just see what works.

**Pull request merged**